dashboard/app/views/levels/show.html.haml

```diff
@@ -22,6 +22,9 @@
   :javascript
     var appOptions = #{app_options.to_json};
   = render partial: 'levels/unplug', locals: {app: @game.app}
+- elsif @level.is_a?(Applab) && app_options['readonlyWorkspace'] && !app_options[:channel]
```

Comment:
I used a mix of strings and symbols in the code I added because when I only used strings, it wasn't finding the keys which were symbols. I am open to suggestions if there is a way to write this better.

I'm not sure how this affects other level types besides Applab. I decided to limit this to Applab to limit the scope of this change, so that it didn't have unintended consequences for other level types. The way this change is structured it will be easy for anyone to add other level types to this condition in the future if they are confident that doing so won't break things.